src/graphql.tsx

```diff
@@ -635,7 +635,10 @@ class ObservableQueryRecycler {
   public recycle (observableQuery: ObservableQuery<any>): void {
     // Stop the query from polling when we recycle. Polling may resume when we
     // reuse it and call `setOptions`.
-    observableQuery.stopPolling();
+    observableQuery.setOptions({
+      fetchPolicy: 'cache-only',
```

Comment:
On second thought, I’m not ready to merge this because if an observable query gets reused the `fetchPolicy` will stay as `cache-only` which is undesirable. Could you write a test to show this doesn’t happen? If it does could you make sure it does not?